### PR TITLE
Bump Documenter to get the new pdf improvements for the manual.

### PR DESCRIPTION
**Pull request by ViralBShah**

### CODE REVIEW

File: doc/Manifest.toml [hunk 1]
```diff
@@ -26,7 +26,7 @@ version = "0.8.6"
 deps = ["ANSIColoredPrinters", "Base64", "Dates", "DocStringExtensions", "IOCapture", "InteractiveUtils", "JSON", "LibGit2", "Logging", "Markdown", "REPL", "Test", "Unicode"]
 git-tree-sha1 = "2c023382ab49c40475fcf59b90ba1c8edd9ff45e"
```
Comment (giordano) [inline doc/Manifest.toml:27]: Don't you need to change the tree hash as well?

Comment (KristofferC) [reply doc/Manifest.toml:27]: See the top of this file:

> # This file is machine-generated - editing it directly is not advised

:)

Comment (ViralBShah) [reply doc/Manifest.toml:27]: Thanks

**Comment by ViralBShah** [comment]
The pdf build of the manual is successful (I get a complete pdf), but I get this error messge. When I look in the directory it asks me to check the `.log` file, I see the pdf

```
NEWS-update.jl  UnicodeData.txt make.jl
➜  doc git:(master) make pdf
/Users/viral/julia/deps/tools/jlchecksum "/Users/viral/julia/deps/srccache/UnicodeData-13.0.0.txt"
cp "/Users/viral/julia/deps/srccache/UnicodeData-13.0.0.txt" UnicodeData.txt
Building PDF documentation.
/Users/viral/julia/usr/bin/julia --startup-file=no --color=yes /Users/viral/julia/doc/make.jl -- pdf linkcheck= doctest= buildroot=/Users/viral/julia texplatform= revise=
Precompiling project...
  6 dependencies successfully precompiled in 28 seconds
[ Info: SetupBuildDirectory: setting up build directory.
[ Info: Doctest: skipped.
[ Info: ExpandTemplates: expanding markdown templates.
[ Info: CrossReferences: building cross-references.
[ Info: CheckDocument: running document checks.
[ Info: Populate: populating indices.
[ Info: RenderDocument: rendering document.
[ Info: LaTeXWriter: creating the LaTeX file.
[ Info: LaTeXWriter: using latexmk to compile tex.
┌ Error: LaTeXWriter: failed to compile tex with latexmk. Logs and partial output can be found in /var/folders/rj/njz8jnv11637flz6vgnb6bz00000gp/T/jl_0pW3p1.
│   exception =
│    failed process: Process(`latexmk -f -interaction=nonstopmode -view=none -lualatex -shell-escape TheJuliaLanguage.tex`, ProcessExited(12)) [12]
│    
└ @ Documenter.Writers.LaTeXWriter ~/julia/doc/deps/packages/Documenter/9rkAd/src/Writers/LaTeXWriter.jl:176
ERROR: LoadError: Compiling the .tex file failed. See logs for more information.
Stacktrace:
  [1] error(s::String)
    @ Base ./error.jl:35
  [2] (::Documenter.Writers.LaTeXWriter.var"#5#8"{Documenter.Documents.Document, Documenter.Writers.LaTeXWriter.LaTeX})()
    @ Documenter.Writers.LaTeXWriter ~/julia/doc/deps/packages/Documenter/9rkAd/src/Writers/LaTeXWriter.jl:159
  [3] cd(f::Documenter.Writers.LaTeXWriter.var"#5#8"{Documenter.Documents.Document, Documenter.Writers.LaTeXWriter.LaTeX}, dir::String)
    @ Base.Filesystem ./file.jl:112
  [4] (::Documenter.Writers.LaTeXWriter.var"#4#7"{Documenter.Documents.Document, Documenter.Writers.LaTeXWriter.LaTeX})(path::String)
    @ Documenter.Writers.LaTeXWriter ~/julia/doc/deps/packages/Documenter/9rkAd/src/Writers/LaTeXWriter.jl:104
  [5] mktempdir(fn::Documenter.Writers.LaTeXWriter.var"#4#7"{Documenter.Documents.Document, Documenter.Writers.LaTeXWriter.LaTeX}, parent::String; prefix::String)
    @ Base.Filesystem ./file.jl:760
  [6] mktempdir (repeats 2 times)
    @ ./file.jl:758 [inlined]
  [7] render(doc::Documenter.Documents.Document, settings::Documenter.Writers.LaTeXWriter.LaTeX)
    @ Documenter.Writers.LaTeXWriter ~/julia/doc/deps/packages/Documenter/9rkAd/src/Writers/LaTeXWriter.jl:102
  [8] runner(#unused#::Type{Documenter.Writers.LaTeXFormat}, fmt::Documenter.Writers.LaTeXWriter.LaTeX, doc::Documenter.Documents.Document)
    @ Documenter.Writers ~/julia/doc/deps/packages/Documenter/9rkAd/src/Writers/Writers.jl:41
  [9] dispatch(::Type{Documenter.Writers.FormatSelector}, ::Documenter.Writers.LaTeXWriter.LaTeX, ::Vararg{Any})
    @ Documenter.Utilities.Selectors ~/julia/doc/deps/packages/Documenter/9rkAd/src/Utilities/Selectors.jl:170
 [10] render(doc::Documenter.Documents.Document)
    @ Documenter.Writers ~/julia/doc/deps/packages/Documenter/9rkAd/src/Writers/Writers.jl:86
 [11] runner(#unused#::Type{Documenter.Builder.RenderDocument}, doc::Documenter.Documents.Document)
    @ Documenter.Builder ~/julia/doc/deps/packages/Documenter/9rkAd/src/Builder.jl:258
 [12] dispatch(#unused#::Type{Documenter.Builder.DocumentPipeline}, x::Documenter.Documents.Document)
    @ Documenter.Utilities.Selectors ~/julia/doc/deps/packages/Documenter/9rkAd/src/Utilities/Selectors.jl:170
 [13] #2
    @ ~/julia/doc/deps/packages/Documenter/9rkAd/src/Documenter.jl:266 [inlined]
 [14] cd(f::Documenter.var"#2#3"{Documenter.Documents.Document}, dir::String)
    @ Base.Filesystem ./file.jl:112
 [15] makedocs(; debug::Bool, format::Documenter.Writers.LaTeXWriter.LaTeX, kwargs::Base.Pairs{Symbol, Any, NTuple{11, Symbol}, NamedTuple{(:build, :modules, :clean, :doctest, :linkcheck, :linkcheck_ignore, :strict, :checkdocs, :sitename, :authors, :pages), Tuple{String, Vector{Module}, Bool, Bool, Bool, Vector{String}, Bool, Symbol, String, String, Vector{Any}}}})
    @ Documenter ~/julia/doc/deps/packages/Documenter/9rkAd/src/Documenter.jl:265
 [16] top-level scope
    @ ~/julia/doc/make.jl:300
in expression starting at /Users/viral/julia/doc/make.jl:300
```

And the log file also shows a successful build:

```
Output written on TheJuliaLanguage.pdf (946 pages, 3991084 bytes).

PDF statistics: 25307 PDF objects out of 26593 (max. 8388607)
 23748 compressed objects within 238 object streams
 8197 named destinations out of 8907 (max. 131072)
 4624 words of extra memory for PDF output out of 10000 (max. 100000000)
```

**Comment by ViralBShah** [comment]
Also, ideally I would like to have a Documenter.jl release with https://github.com/JuliaDocs/Documenter.jl/pull/1785

**Comment by mortenpi** [comment]
@ViralBShah @inkydragon FYI: a new Documenter version has been registered now.

**Review by mortenpi — APPROVED** [review]

**Pull request merged**